frontend/src/components/TreeViewer/TreeViewer.jsx

```diff
@@ -441,6 +454,7 @@ export const TreeViewer = ({ data, heading }) => {
       })
       .then((res) => {
         setNodeInfoContent(res);
+        setNodeInfoName(clickedNodeFromMapping.name);
```

Comment:
Question: why do we need this line, wasn't the name appearing at the top of the infoviewer already?

Reply:
nvm I understand...man we really need a global application state lol

Reply:
What would a global application state entail? Like a single object that has properties that we update?

Reply:
yup, exactly, that is correct. But we can do that in the next phase